[BUG] Argument format error in xxd command (ush_cmd_xxd.c)

Compiling with arm-none-eabi GCC 8.3.1 (target is RP2040) - in the hex printing routine of `ush_cmd_xxd.c` there is a size mismatch of the formatted string and the item passed to `sprintf()`.

Build output:
```
src/commands/ush_cmd_xxd.c: In function 'ush_buildin_cmd_xxd_service':
src/commands/ush_cmd_xxd.c:80:64: error: format '%X' expects argument of type 'unsigned int', but argument 3 has type 'long unsigned int' [-Werror=format=]
                          sprintf(self->desc->output_buffer, "%08X: ", (uint32_t)self->process_index_item);
                                                              ~~~^     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                              %08lX
```

As shown above the error can be fixed by using `%08lX` (long) format rather than `%08X`. I think perhaps this issue does not arise when compiling for POSIX, but does on ARM?

--- a/src/src/commands/ush_cmd_xxd.c
+++ b/src/src/commands/ush_cmd_xxd.c
@@ -77,7 +77,7 @@ bool ush_buildin_cmd_xxd_service(struct ush_object *self, struct ush_file_descri
         case USH_STATE_PROCESS_SERVICE: {
                 switch (self->process_index) {
                 case 0:
-                        sprintf(self->desc->output_buffer, "%08X: ", (uint32_t)self->process_index_item);
+                        sprintf(self->desc->output_buffer, "%08lX: ", self->process_index_item);
                         ush_write_pointer(self, self->desc->output_buffer, self->state);
                         self->process_index = 1;
                         break;

